### PR TITLE
Support Advertised prefixes

### PR DESCRIPTION
**Pull request by jgomezve**

### CODE REVIEW

**Comment by jgomezve** [comment]
The DME model mentions the attribute `evpn`, which is also configurable via cli `network x.x.x.x evpn`, but the API does not recognize this attribute. It is also not shown when using the NX-API Sandbox converter

File: gen/definitions/bgp_advertised_address.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      - ipv6-mvpn
+      - l2vpn-vpls
+      - ipv4-mdt
+    default_value: 'ipv4-ucast'
+    example: 'ipv4-ucast'
+  - nxos_name: addr
+    tf_name: prefix
+    type: String
+    id: true
+    description: 'The next-hop address tracking delay timer for critical next-hop reachability routes.'
```
Comment (danischm) [inline gen/definitions/bgp_advertised_address.yaml:53]: I believe this needs to be updated.

File: gen/definitions/bgp_advertised_address.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+name: BGP Advertised Prefix
+class_name: bgpAdvPrefix
+dn: sys/bgp/inst/dom-[%s]/af-[%s]/prefix-[%s]
+ds_description: This data source can read the BGP (VRF) advertise prefix configuration.
+res_description: This resource can manage the BGP (VRF) advertise prefix configuration.
+doc_path: Routing%20and%20Forwarding/bgp:AdvPrefix/
+doc_category: BGP
+parents:
+  - BGP Address Family
```
Comment (danischm) [inline gen/definitions/bgp_advertised_address.yaml:10]: We should then also update the "BGP Address Family" definition to list this as a child.

File: gen/definitions/bgp_advertised_address.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  - nxos_name: addr
+    tf_name: prefix
+    type: String
+    id: true
+    description: 'The next-hop address tracking delay timer for critical next-hop reachability routes.'
+    example: '192.168.1.0/24'
+  - nxos_name: rtMap
+    tf_name: route_map
+    type: String
+    description: 'Route map to modify attributese.'
```
Comment (danischm) [inline gen/definitions/bgp_advertised_address.yaml:58]: typo

**Review by danischm — CHANGES_REQUESTED** [review]

**Review by danischm — APPROVED** [review]

**Pull request merged**